gen/js/chromestatus-openapi/package.json

```diff
@@ -16,6 +16,6 @@
     "prepare": "npm run build"
   },
   "devDependencies": {
-    "typescript": "^5.3"
+    "typescript": "^4.0"
```

Comment:
Nothing wrong here.

But I am very curious as to how this was 5.3 beforehand haha. I see the [mustache](https://github.com/OpenAPITools/openapi-generator/blob/2129b15c8f0598126a7a259106f7018bd302eef2/modules/openapi-generator/src/main/resources/typescript-fetch/package.mustache#L33) file pins it to ^4.0